### ISSUE TITLE
[A11y] 4.1.2 Name, Role, Value – ARIA Dialog Has No Accessible Name

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1752]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/66 for more details

**Comment by dalonsoa** [comment]
I've addressed this one in #1772
